install/lib/class.FroxlorInstall.php

```diff
@@ -66,7 +66,8 @@ class FroxlorInstall {
 	private $_languages = array(
 			'german' => 'Deutsch',
 			'english' => 'English',
-			'french' => 'Français'
+			'french' => 'Français',
+			'polish' => 'Polish'
```

Comment:
the value (right side) of the array-item should be the language-name in the corresponding language, not english, in this case i guess "Polski"